AcadeMeData/models.py

```diff
@@ -239,3 +241,11 @@ def get_course_by_name(name):
             return course
         except Course.DoesNotExist:
             return None
+
+    def course_belongs(self, university, degree):
+        """
+        returns if course belongs to this degree in this university
+        """
+        if degree in self.degree.all() and university == self.university:
```

Comment:
Using [Truthy and Falsy](https://www.geeksforgeeks.org/truthy-vs-falsy-values-in-python/) values is considered more pythonic than returning True\False like this.

try to modify the function so it'll turn into a one-liner :)

Reply:
Turned it to a one-liner @Omeramsc